core/src/main/java/org/opensearch/sql/data/model/AbstractExprValue.java

```diff
@@ -34,13 +29,8 @@ public int compareTo(ExprValue other) {
   }
 
   /**
-   * The customize equals logic.
-   * The table below list the NULL and MISSING handling logic.
-   * A       B       A == B
-   * NULL    NULL    TRUE
-   * NULL    MISSING FALSE
-   * MISSING NULL    FALSE
-   * MISSING MISSING TRUE
+   * The customize equals logic. The table below list the NULL and MISSING handling logic. A B A ==
```

Comment:
Also needs `<pre>` tag

Reply:
Addressed by adding table in [d455533](https://github.com/Bit-Quill/opensearch-project-sql/pull/312/commits/d4555331f77ba3c50c9a38627cc3b35550ebfda7)